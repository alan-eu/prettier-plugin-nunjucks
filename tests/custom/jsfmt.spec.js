run_spec(__dirname, ["nunjucks"], {
  blockTags: ['remote'],
  forkTags: ['error']
});
