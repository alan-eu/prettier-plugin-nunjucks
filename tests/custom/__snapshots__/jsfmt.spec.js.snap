// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom.njk 1`] = `
====================================options=====================================
parsers: ["nunjucks"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
{% remote "/stuff" %}
  This content will be replaced with the content from /stuff
      {% error %}
            There was an error fetching /stuff
          {% endremote %}

=====================================output=====================================
{% remote "/stuff" %}
  This content will be replaced with the content from /stuff
{% error %}
  There was an error fetching /stuff
{% endremote %}

================================================================================
`;
